index.html

```diff
@@ -63,9 +65,9 @@ <h3>About Me</h3>
                 <button class="btn">Read More</button>
             </div>
      </div>
-     <div class="rightside md-col-6">
-
-        <img class="photo"src="https://coachtrainingworld.com/wp-content/uploads/2015/10/start_life_coaching_business.jpg" alt="life-coach-img">
+     <div class="rightside col-12 md-col-6">
+        
+        <!--<img class="photo"src="https://coachtrainingworld.com/wp-content/uploads/2015/10/start_life_coaching_business.jpg" alt="life-coach-img">-->
```

Comment:
If you are not using code then you should delete it, comments are for communication

Reply:
done